### PR TITLE
Allow making use of Ansible-vault to encrypt and decrypt sensitive certs and keys

### PR DESCRIPTION
**Pull request by IzakEygelaar**

### CODE REVIEW

**Comment by IzakEygelaar** [comment]
Also locking down permissions on these files to remove world readability

**Comment by IzakEygelaar** [comment]
> The copy module already sets `decrypt` to yes by default but it may be a good idea to explicitly set the parameter in case defaults change in the future.

Good point @alessfg, forgot about the default behaviour is `decrypt: yes` :)

> For parity purposes could you also set the `decrypt` parameter for the task responsible for uploading NGINX Plus certs and keys in `tasks/plus/setup-license.yml`?

Coolios, will do!

**Comment by IzakEygelaar** [comment]
Added explicit `decrypt: yes` and removed world readability from license cert and key

File: tasks/plus/setup-license.yml [hunk 1]
```diff
@@ -8,6 +8,8 @@
   copy:
     src: "{{ item }}"
     dest: /etc/ssl/nginx
+    mode: 0640
```
Comment (alessfg) [inline tasks/plus/setup-license.yml]:
```suggestion
```

Limiting permissions for the license causes some issues with the installation process. Better to get rid of it for the time being.

Comment (IzakEygelaar) [reply tasks/plus/setup-license.yml]: Removed for now.

**Pull request merged**